2018AsianGamesApp/app/src/main/java/com/example/mrtbkwjn/a2018asiangamesapp/EventAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.mrtbkwjn.a2018asiangamesapp;
+
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
```

Comment:
There are a lot of empty spaces here, which makes reading it a bit confusing. One empty line should suffice

Reply:
Noted!